core/src/impl/Kokkos_Default_Graph_Impl.hpp

```diff
@@ -82,7 +82,7 @@ struct GraphImpl : private ExecutionSpaceInstanceStorage<ExecutionSpace> {
   // <editor-fold desc="required customizations"> {{{2
 
   template <class NodeImpl>
-  void add_node(std::shared_ptr<NodeImpl> const& arg_node_ptr) {
+  void add_node(std::shared_ptr<NodeImpl> arg_node_ptr) {
```

Comment:
There are more cases of passing `std::shared_ptr` by value or `const&`. If we care about unifying the approach that should be done in a different pull request.
```suggestion
  void add_node(std::shared_ptr<NodeImpl> const& arg_node_ptr) {
```

Reply:
I think this is the only "not aligned occurrence". All other `add_node` take by value explicitly it needed.

Reply:
If you really feel strong about it, I can revert this change :wink: